File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/NoiTru_TraThuocService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class NoiTru_TraThuocService : BaseService<NoiTru_TraThuoc, INoiTru_TraThuocRepository>
    , INoiTru_TraThuocService
    {
    private readonly INoiTru_TraThuocRepository _NoiTru_TraThuocRepository;
    public NoiTru_TraThuocService(INoiTru_TraThuocRepository NoiTru_TraThuocRepository) : base(NoiTru_TraThuocRepository)
    {
    _NoiTru_TraThuocRepository = NoiTru_TraThuocRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/NoiTru_TraThuoc.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NoiTru_TraThuoc } from './NoiTru_TraThuoc.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NoiTru_TraThuocService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'NoiTru_TraThuoc_Id', 'SoPhieu', 'BenhAn_Id', 'LuuTru_Id', 'NgayTra', 'NguoiNhan_Id', 'GhiChu', 'TrangThai', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'DienGiaiNghiepVuPhatSinh', 'ThoiGianTra', 'ChungTu_Id'];

    List: NoiTru_TraThuoc[] | undefined;
    ListFilter: NoiTru_TraThuoc[] | undefined;
    FormData!: NoiTru_TraThuoc;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NoiTru_TraThuoc";
    }
}

